docs/resources/powerpack.md

```diff
@@ -28,6 +29,21 @@ Provides a Datadog powerpack resource. This can be used to create and manage Dat
 
 - `id` (String) The ID of this resource.
 
+<a id="nestedblock--layout"></a>
+### Nested Schema for `layout`
+
+Required:
+
+- `height` (Number) The height of the widget.
+- `width` (Number) The width of the widget.
+- `x` (Number) The position of the widget on the x (horizontal) axis. Should be greater than or equal to 0.
+- `y` (Number) The position of the widget on the y (vertical) axis. Should be greater than or equal to 0.
+
+Optional:
+
+- `is_column_break` (Boolean) Whether the widget should be the first one on the second column in high density or not. Only for the new dashboard layout and only one widget in the dashboard should have this property set to `true`.
```

Comment:
Would it be possible to clarify what `the new dashboard layout` is referencing?

Reply:
@buraizu This description is generated from an inherited dashboard widget layout schema, so we currently have the same language for dashboard.md [[src](https://github.com/DataDog/terraform-provider-datadog/blob/master/docs/resources/dashboard.md#nested-schema-for-widgetgroup_definitionwidgetwidget_layout)]. What do you think about me addressing this in a follow up where I fix the language in the original dashboard resource?

Reply:
@dashashifrina Thanks for following up to confirm that. As far as I can tell, there isn't any reference to `the new dashboard layout` anywhere else in the documentation, so a bit of clarification here would indeed be helpful. Let me know if there is anything I can do to assist.